extensions/firebase-manage/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  extends: ["@raycast"],
+  rules: {
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/no-unused-vars": "off",
+    "@raycast/prefer-title-case": "off"
+  }
+}; 
```

Comment:
We should keep all extensions' code styles consistent. Please remove your personal preference ESLint settings 😊.